File: 2740. Find the Value of the Partition/Solution.java
class Solution {
    public int findValueOfPartition(int[] nums) {
        Arrays.sort(nums);
        int low = 1000000007, n = nums.length;
        for (int i = 1; i < n; i++) {
            low = Math.min(low, nums[i] - nums[i - 1]);
        }
        return low;
    }
}

File: 2740. Find the Value of the Partition/Solution.cpp
class Solution {
public:
    int findValueOfPartition(vector<int>& nums) {
        sort(begin(nums), end(nums));
        int low = 1000000007, n = nums.size();
        for (int i = 1; i < n; i++) {
            low = min(low, nums[i] - nums[i - 1]);
        }
        return low;
    }
};
